### DISCUSSION TITLE
Integration of Xiaomi Smart watch s1 active

### DISCUSSION THREAD

**Discussion by maximoff1968** [NightscoutFoundation/xDrip#3304] [Q&A]
Hello, please tell me, is there any software planned for XIAOMI SMART WATCH S1 ACTIVE to display a glucose graph?

**Comment by Navid200** [comment]
Please tell us about the watch.
Is it WearOS?  if yes, we currently don't have any developer taking care of WearOS unfortunately.
Or, is it a full Android watch?

**Reply by maximoff1968** [reply]
Hello, thanks for your message.  Yes, the operating system is based on Android.  System: MIUI WATCH 1.0 Version: 1.4.174 MODEL: M2116W1

**Reply by Navid200** [reply]
According to this, it uses proprietary OS: https://www.gsmarena.com/xiaomi_watch_s1_active-11422.php

We try to provide xDrip for all Android phones.  Yet, there are phones that struggle because of the way the manufacturer has messed with Bluetooth stack or keeps killing xDrip in the background.
There is no practical way for us to support a proprietary OS.
